src/main/java/seedu/address/model/contact/UniqueContactList.java

```diff
@@ -101,6 +101,26 @@ public void remove(Contact toRemove) {
         }
     }
 
+    /**
+     * Verifies that the {@code editedContact} is valid and does not contain any repeated names
```

Comment:
Similar typo to the previous :)

Reply:
Have corrected it! Thanks for spotting :)